slo-monitor/src/monitors/pod_monitor.go

```diff
@@ -175,9 +186,19 @@ func (pm *PodStartupLatencyDataMonitor) Run(stopCh chan struct{}) error {
 	eventController := NewWatcherWithHandler(
 		&cache.ListWatch{
 			ListFunc: func(options metav1.ListOptions) (runtime.Object, error) {
-				options.FieldSelector = eventSelector
-				// This call is paginated by cache.Reflector.
-				return pm.kubeClient.CoreV1().Events(v1.NamespaceAll).List(options)
+				// slo monitor is not using list of events.
```

Comment:
I would add more context, like: NewWatcherWithHandler internally uses NoStore store that ignores all relist results, so here we do List call only to get the current resource version.